azure/activity_logs_monitoring/index.js

```diff
@@ -29,6 +29,9 @@ const DD_SERVICE = process.env.DD_SERVICE || 'azure';
 const DD_SOURCE = process.env.DD_SOURCE || 'azure';
 const DD_SOURCE_CATEGORY = process.env.DD_SOURCE_CATEGORY || 'azure';
 
+const MAX_RETRIES = 4; // max number of times to retry a single http request
+const RETRY_INTERVAL = 1000; // amount of time (milliseconds) to wait before retrying request, doubles after every retry
```

Comment:
The function is a bit sensitive to latency so what if we start at like 250 ms and double each time?